src/main/java/net/earthcomputer/clientcommands/mixin/CommandSuggestorAccessor.java

```diff
@@ -0,0 +1,17 @@
+package net.earthcomputer.clientcommands.mixin;
+
+import java.util.List;
+import net.minecraft.client.gui.screen.CommandSuggestor;
+import net.minecraft.text.Style;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(CommandSuggestor.class)
+public interface CommandSuggestorAccessor {
+
+    @Accessor
+    static List<Style> getHIGHLIGHT_FORMATTINGS() {
```

Comment:
This is ugly, could you put the name in the accessor annotation instead?

Reply:
Fixed in 237b11e09568d4bb16bf97f5180d2f3ceb7c1e7e